Library/.rubocop.yml

```diff
@@ -28,7 +28,7 @@ FormulaAuditStrict/Options:
   Enabled: true
 
 NewFormulaAudit/Options:
-  Enabled: false
+  Enabled: true 
```

Comment:
@MikeMcQuaid One way that works involves some redundant configuration. (Redundant because we are doing similar thing in `brew style` , but it has to be redundant because `brew` and `rubocop` run in different processes and aren't as such coupled and editor integration is independent of `brew`)

In sublime, we can add `args` line in Sublime Linter User Settings (found in `Menu->Preferences->Package Settings->SublimeLinter->Settings - User`) so final `rubocop` section should look like below 

```
"rubocop": {
       "@disable": false,
       "args": ["--except", "FormulaAuditStrict, NewFormulaAudit"],
       "excludes": []
}
```
If we are planning to go by this method, It would be better to update our docs to have a section on editor integration.